### PR TITLE
[skip-ci][DF] Fixes of typos and grammar mistakes in the df007 tutorials

### PR DESCRIPTION
**Pull request by j-mathe**

### CODE REVIEW

**Comment by ikabadzhov** [comment]
Thank you very much @j-mathe. There is also *this* that is outdated:
```python
branchList = ROOT.vector('string')()
for branchName in ["b1", "b1_square", "b2_vector"]:
    branchList.push_back(branchName)
d2.Snapshot(treeName, outFileName, branchList)
```
It can directly be: `d2.Snapshot(treeName, outFileName, ["b1", "b1_square", "b2_vector"])`

Can you plese update it. And since you are touch the source code, you should also remove the `[skip-ci]`

File: tutorials/dataframe/df007_snapshot.C [hunk 1]
```diff
@@ -36,14 +36,14 @@ int df007_snapshot()
...
-   // We read the tree from the file and create a RDataFrame.
+   // We read the tree from the file and create a RDataFrame
    ROOT::RDataFrame d(treeName, fileName);
 
    // ## Select entries
    // We now select some entries in the dataset
    auto d_cut = d.Filter("b1 % 2 == 0");
    // ## Enrich the dataset
-   // Build some temporary columns: we'll write them out
+   // Build some temporary columns: We'll write them out
```
Comment (eguiraud) [inline tutorials/dataframe/df007_snapshot.C:46]: as I commented in some other PR, English does not use caps after `:`

Comment (j-mathe) [reply tutorials/dataframe/df007_snapshot.C]: Sorry, I missed that. I fixed it now.

Comment (ikabadzhov) [reply tutorials/dataframe/df007_snapshot.C]: thank you. note that is comment is applicable in few other places as well

**Review by eguiraud — APPROVED** [review]
great, thanks!

**Pull request merged**